source/views/dictionary/list.js

```diff
@@ -41,6 +57,12 @@ export class DictionaryView extends React.Component {
     route: React.PropTypes.object.isRequired,
   }
 
+  state: {
+    results: {[key: string]: Object},
```

Comment:
So instead of `Object`, you should describe the shape of the object that it holds.